pallets/avn/src/lib.rs

```
@@ -398,13 +406,15 @@ impl<T: Config> OneSessionHandler<T::AccountId> for Pallet<T> {
         trace!("Avn pallet new session entrypoint");
         // Update the list of validators if it has changed
         let mut disabled_avn_validators: Vec<T::AccountId> = vec![];
-        let mut active_avn_validators: Vec<Validator<T::AuthorityId, T::AccountId>> = vec![];
+        let mut active_avn_validators = BoundedVec::default();
 
         validators.for_each(|x| {
             if T::DisabledValidatorChecker::is_disabled(x.0) {
                 disabled_avn_validators.push(x.0.clone());
             } else {
-                active_avn_validators.push(Validator::new(x.0.clone(), x.1));
+                active_avn_validators
+                    .try_push(Validator::new(x.0.clone(), x.1))
+                    .map_err(|_| DispatchError::Other("Maximum Validators reached"));
```

we ignore the error here